scripts/o3de/o3de/register.py

```diff
@@ -614,6 +614,13 @@ def register_repo(json_data: dict,
 
     return 1
 
+def register_remote_repo(json_data: dict,
+                  repo_uri: str,) -> int:
```

Comment:
There is a trailing comma in the method signature that should be removed.
```suggestion
                  repo_uri: str) -> int:
```